src/blog-posts/q4-2024-feature-roundup.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Q4 Feature Roundup: Gasless claiming, Filecoin, Embeddable support buttons'
```

Comment:
I think we should change the name to remove 'feature' cos it might not always be as feature heavy in future. Sth like the 'quarterly catch-up'. I need to ponder a few more ides for it tbh